server/BUILD

```diff
@@ -94,12 +94,20 @@ native_java_libraries(
     visibility = ["//visibility:public"],
 )
 
+genrule(
+    name = "prepare-server-directories",
+    srcs = [":config"],
+    # TODO: can we find a way to create a directory without including an empty file in the outs?
+    cmd = "cp $(location :config) $(@D)/conf/ && touch $(@D)/data/empty",
+    outs = ["conf/config.yml", "data/empty"]
+)
+
```

Comment:
this is the new rule that prepares the directory structure expected by the server binary - copies the `config.yml` and create the `data` directory